common/vcenter_manage_vds_portgroup.yml

```diff
@@ -0,0 +1,83 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Create or remove a vSphere Distributed Port Group
+# Parameters:
+#   vds_name: The vSphere Distributed Switch name where the port group is.
+#   vds_portgroup_name: The distributed port group name.
+#   vds_portgroup_state: The distributed port group state: present or absent.
+#   vds_port_number: The number of ports the distributed port group should contain.
```

Comment:
This parameter can also have a default value '4'

Reply:
Updated.